src/main/java/biz/wgc/fwplan/constants/ApplicationSettings.java

```diff
@@ -28,11 +28,11 @@ public static void setSettings() throws IOException {
 		InputStream inStream = new FileInputStream(APP_PROPERTIES);
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer

**Problem**
This line of code contains a resource that might not be closed properly. Resource leaks can cause your system to slow down or crash.

**Fix**
Consider closing the following resource: *inStream*. The resource is referenced in statements at the following line: 29. The resource closure statement is at line: 37. There are other execution paths that do not contain closure statements, e.g., when exception is thrown by Properties.load. Either a) close *inStream* in a try-finally block or b) close the resource by declaring *inStream* in a try-with-resources block.

**More info**
[View resource management guidelines at oracle.com](https://www.oracle.com/technetwork/articles/java/trywithresources-401775.html) (external link).